modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -39,14 +39,11 @@
  * <p>
  * An implementation of {@code ObservableValue} may support lazy evaluation,
  * which means that the value is not immediately recomputed after changes, but
- * lazily the next time the value is requested. All bindings and properties in
- * this library support lazy evaluation.
+ * lazily the next time the value is requested (see note 1 in {@code implSpec}).
```

Comment:
The term "implSpec" doesn't show up in the generated documents, so maybe change it to something like:

    See note 1 in the "Implementation Requirements"